frontend/packages/ceph-storage-plugin/integration-tests/tests/4-pools/multiple-pool.scenario.ts

```diff
@@ -0,0 +1,100 @@
+import { execSync } from 'child_process';
+import { NS, CLUSTER_STATUS, SECOND } from '../../utils/consts';
+import {
+  createStorageClass,
+  allowExpand,
+  poolDropdown,
+  poolDropdownButton,
+  createPoolDropdown,
+  progressStateCheck,
+  createPool,
+  poolForm,
+  poolStartStateCheck,
+} from '../../views/multiple-pool.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { browser, element, protractor } from 'protractor';
+
+const cephValue = JSON.parse(execSync(`kubectl get cephCluster -n ${NS} -o json`).toString());
+const cephHealth = cephValue.items[0];
+
+describe('Check availability of Ceph cluster', () => {
+  if (!cephHealth) {
+    it('Should state that Ceph cluster doesnt exist', () => {
+      expect(cephHealth).toBeUndefined();
+    });
+  } else {
+    it('Should state that Ceph cluster exist', () => {
+      expect(cephHealth).toBeDefined();
+    });
+  }
+});
+
+if (cephHealth && cephHealth.status.phase && cephHealth.status.phase !== CLUSTER_STATUS.READY) {
+  describe('Check for pool creation if ceph cluster is in progress', () => {
+    beforeAll(async () => {
+      createStorageClass('openshift-storage.rbd.csi.ceph.com');
```

Comment:
`createStorageClass` method is not creating a storage class, it should named proper. Maybe `prepareStorageClass` ?